versioned_docs/version-v0.6.0/installation/openyurt-experience-center/overview.md

```diff
@@ -2,37 +2,37 @@
 title: OpenYurt Experience Center Introduction
 ---
 
-> [体验平台 beta 版本](http://139.224.236.157/)已经开放，欢迎大家使用反馈。域名正在申请中，后续正式版本将使用`lab.openyurt.io`作为域名。
+> [Experience Center-Beta](http://139.224.236.157/) has been ready, feel free to use and submit feedback. The domain name is under applying, and the subsequent release version will use `lab.openyurt.io` as the domain name.
 
-## 总体介绍
+## Overview
 
-OpenYurt 作为边缘云原生项目,涉及到边缘计算和云原生两个领域,许多边缘计算的开发者不熟悉云原生相关的知识。为了降低 OpenYurt 的使用门槛，帮助更多地开发者快速上手 OpenYurt，社区推出了 OpenYurt 体验中心这一平台。想要尝试 OpenYurt 能力的新用户无需从头搭建 OpenYurt 集群, 只需要在平台上申请一个测试账号，即可马上拥有一个可用的 OpenYurt 集群。
+As an edge cloud native project, OpenYurt involves both edge computing and cloud native areas, many edge computing developers are not familiar with cloud native related technologies. In order to lower the threshold of using OpenYurt and help more developers get started with OpenYurt quickly, our community has launched a platform called OpenYurt Experience Center. Instead of building an OpenYurt cluster from scratch, new users who want to try out OpenYurt's features can simply apply for a test account on the platform and have an available OpenYurt cluster right away.
 
 ![](../../../../static/img/docs/installation/openyurt-experience-center/register_blank.png)
 
-## 体验中心的功能
+## Features
 
-体验中心会为每个[注册用户](./user.md)分配一个 OpenYurt 集群。（注意这个集群并不是一个功能完整的 K8s 集群，用户只能在自己的 namespace 下活动，详情请参见体验中心的架构介绍）
+Platform will assign an OpenYurt cluster to each [registered user](./user.md). (Note that this cluster is not a full-featured K8s cluster and this means users can only be active under their own namespace; see the Experience Center's architecture for details)
 
-对于该集群，体验中心提供了两种使用方式：
+Two types of usage are offered:
```

Comment:
how about `OpenYurt Experience Center has provided two ways to use OpenYurt cluster:`